### PR TITLE
feat: add URL to start payment

### PR DESCRIPTION
**Pull request by JaZo**
## How-to-demo
1. Make sure you have the Mollie key;
2. Start `ddev share`;
3. Go to https://[NGROK HOST]/transactions/[TENANT ID]/start/[MD5 OF EMAIL]?mass=[MASS IN KG]&distance=[DISTANCE IN KM]&orderId=[WEBSHOP ORDER ID];
4. See the account is created;
5. See the transaction is created;
6. See you're redirected to Mollie;
7. Finish the payment.

### CODE REVIEW

**Comment by JaZo** [comment]
:warning: The following controllers contain actions outside of the 7 resource actions (`index`, `create`, `store`, `show`, `edit`, `update`, `destroy`). For more details, review [the docs](https://laravel.com/docs/controllers#resource-controllers) or watch [Cruddy by Design](https://www.youtube.com/watch?v=MF0jFKvS4SI) to see if you may rework these into *resource controllers*.

- [ ] app/Http/Controllers/PaymentController.php

**Comment by JaZo** [comment]
:information_source: Now with type hints in your code, defining types within PHP DocBlocks is redundant. Laravel has removed all of the `@param` and `@return` tags from its DocBlocks where types are defined with PHP. Similarly, Shift [removed these tags](https://github.com/swisnl/dlf-hackathon-2023/pull/31/commits/440a082dacdb1540651b565d5220cbb268219ce2?w=1) from any DocBlock where the code now has _equivalent_ type hints.

**Comment by JaZo** [comment]
:information_source: While the _Laravel Fixer_ performs automation to adopt the latest Laravel conventions, Shift understands developers have different preferences. All of Shift's automation is done in nice, atomic commits. This makes it easier to undo any of the changes Shift makes using `git revert`.

File: README.md [hunk 1]
```diff
@@ -18,3 +18,7 @@
 ### Filament
 
 This application uses [Filament](https://filamentphp.com/) as its backend administration interface. You can find the login page at `/admin`
+
+### URL
+
+https://778e-45-94-180-186.ngrok-free.app/transactions/9a4bc3e7-6bbf-408a-be6a-3a21d19f8004/start/6b6c735a575cc515f5b1619cc7949ed2?mass=10&distance=100&orderId=2023-AA22423
```
Comment (Rocksheep) [inline README.md:24]: Very interesting, my guess is that it's your local URL?

**Review by Rocksheep — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JaZo) [reply README.md:24]: It was yes, but not anymore.

**Pull request merged**